Cannot run the local devserver for helloworld-gae-javasdk-tools

Hello,

I tried to follow the steps in the readme (`mvn clean appengine:devserver`) and I saw this error:
```
[ERROR] Could not find goal 'devserver' in plugin com.google.cloud.tools:appengine-maven-plugin:2.3.0 among
 available goals cloudSdkLogin, deploy, deployAll, deployCron, deployDispatch, deployDos, deployIndex, 
deployQueue, genRepoInfoFile, help, run, stage, start, stop -> [Help 1]
```

If I use `appengine:run` I get this error:
```
appengine:run is only available for appengine-web.xml based projects
```

--- a/appengine-standard-java8/helloworld-gae-javasdk-tools/README.md
+++ b/appengine-standard-java8/helloworld-gae-javasdk-tools/README.md
@@ -46,7 +46,7 @@ version. Change this field to specify a specific version name.
 ## Maven
 ### Running locally
 
-    mvn clean appengine:devserver
+    mvn clean appengine:start
 
 To use vist: http://localhost:8080/
 
@@ -54,7 +54,7 @@ To use vist: http://localhost:8080/
 
 Update `src/main/webapp/WEB-INF/appengine-web.xml` `<application>` tag with the Project ID.
 
-    mvn clean appengine:update
+    mvn clean appengine:deploy
 
 To use vist:  https://YOUR-PROJECT-ID.appspot.com
 

--- a/appengine-standard-java8/helloworld-gae-javasdk-tools/pom.xml
+++ b/appengine-standard-java8/helloworld-gae-javasdk-tools/pom.xml
@@ -122,7 +122,7 @@ limitations under the License.
     <plugins>
       <!-- [START gaejavaplugin] -->
       <plugin>
-        <groupId>com.google.appengine</groupId>
+        <groupId>com.google.cloud.tools</groupId>
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.3.0</version>
         <configuration>

